apps/frontend/src/modules/surveys/SurveyProvider.tsx

```diff
@@ -296,7 +299,11 @@ export const SurveyProvider = ({ children }: SurveyProviderProp) => {
                         errors: [],
                     },
                 },
-
+                onCompleted: (data) => {
+                    capture(AnalyticsEnum.UPDATE_SURVEY_STATUS, {
```

Comment:
@afolasope I think we should only capture `UPDATE_SURVEY_STATUS` when `input.status` has a value and we capture `UPDATE_SURVEY` when it doesn't.

We can also determine `feature` for `UPDATE_SURVEY` events from the input, so when there is `input.themeId`, `feature` can be `"Change theme"`, when we have `input.name`, `feature` is `"Rename Survey"`, and when there is `input.settings`, it could be `"Update survey settings"`.

What do you think?